generated/googleapis/lib/acceleratedmobilepageurl/v1.dart

```diff
@@ -117,8 +117,8 @@ class AmpUrl {
   /// The AMP URL pointing to the publisher's web server.
   core.String ampUrl;
 
-  /// The [AMP Cache URL](/amp/cache/overview#amp-cache-url-format) pointing to
-  /// the cached document in the Google AMP Cache.
+  /// The \[AMP Cache URL\](/amp/cache/overview#amp-cache-url-format) pointing
+  /// to the cached document in the Google AMP Cache.
```

Comment:
Hmm, I suppose we could try to resolve this relative to: `documentationLink` (`https://developers.google.com/amp/cache/`), which would likely work because it starts with a slash.. So it would become `https://developers.google.com/amp/cache/overview#amp-cache-url-format`.

On the flip side the public docs says nothing about there being markdown in the `description`:
https://developers.google.com/discovery/v1/reference/apis#resource

In any case, this is a massive improvement over broken links :D

Reply:
Ooo! I didn't even consider that. We'd have to pipe the base url through. Doable, though.